app/react/kubernetes/components/KubernetesEventsDatatable/columns/kind.tsx

```diff
@@ -0,0 +1,9 @@
+import { columnHelper } from './helper';
+
+export const kind = columnHelper.accessor(
+  (event) => event.involvedObject.kind,
+  {
+    header: 'Kind',
+    cell: ({ row: { original: event } }) => event.involvedObject.kind,
```

Comment:
see comment on app/react/kubernetes/components/KubernetesEventsDatatable/columns/message.tsx

this can be removed